src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -326,6 +326,7 @@ public String call() throws RuntimeException {
             result.append(min).append(" Name:    ").append(System.getProperty("java.vm.name")).append("\n");
             result.append(min).append(" Vendor:  ").append(System.getProperty("java.vm.vendor")).append("\n");
             result.append(min).append(" Version: ").append(System.getProperty("java.vm.version")).append("\n");
+            result.append(min).append(" Bit size: ").append(System.getProperty("sun.arch.data.model")).append("\n");
```

Comment:
Or if the system property is `null` then don't add the line... but whatever you do don't say `Bit size: null`